apps/server/src/modules/lesson/service/lesson-copy.service.ts

```diff
@@ -236,8 +236,6 @@ export class LessonCopyService {
 					status: CopyStatusEnum.PARTIAL,
 				};
 				if (nexboardContent) {
-					copiedContent.push(nexboardContent);
-				} else {
 					nexboardStatus.status = CopyStatusEnum.FAIL;
```

Comment:
I would say fail is not a correct status. Is on purpose, so NOT_DOING is more correct.
(However, one must also test this in the front-end, to make sure this kind of message is shown correctly)